arch/cortex-m7/src/mpu.rs

```diff
@@ -0,0 +1,695 @@
+//! Implementation of the memory protection unit for the Cortex-M3 and
+//! Cortex-M4.
+
+use core::cell::Cell;
```

Comment:
The difference between them is the number of mpu regions used (16 for Cortex-M7 instead of just 8). For some weird reasons the file was not updated into the repository.